examples/helpers/mail/MultipleEmailsMultipleRecipients.java

```diff
@@ -0,0 +1,51 @@
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.helpers.mail.objects.Personalization;
+
+import java.io.IOException;
+
+public class MultipleEmailsMultipleRecipients {
+
+  public static void main(String[] args) throws IOException {
+    final Mail mail = new Mail();
+
+    mail.setFrom(new Email("test@example.com", "Example User"));
+    mail.setSubject("Sending with Twilio SendGrid is Fun");
+    mail.setTemplateId("d-12345678901234567890123456789012");
```

Comment:
Nice!

My last suggestion would be to include the sample template, [like here](https://github.com/sendgrid/sendgrid-csharp/blob/master/USE_CASES.md#transactional-templates). Not including that example in the past caused a bit of confusion on how to setup the template, particularly the subject.

Reply:
I'd prefer to link to a doc that explains this since it's not specific to this repo.

Reply:
Maybe this is the closest thing? https://sendgrid.com/docs/ui/sending-email/how-to-send-an-email-with-dynamic-transactional-templates/

Reply:
Updated.